### ISSUE TITLE
Beacon Stations softlock the sub

### ISSUE THREAD

**Issue by Rehexr** [Regalis11/Barotrauma#10257]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

This has happened multiple times now. Seems like activiating a beacon station with the sub on top or near it will completely get it stuck and unable to move by any means, even if it should. Can only be fixed by using teleportsub to move it elsewhere, away from the beacon. This has happened regardless of which beacon variant it was. 
(Please mind that the sub is fully operational here, yet it still cant move)

https://user-images.githubusercontent.com/104232152/200140718-30354dab-5140-4e71-961a-e757ebb0d32a.mp4

### Reproduction steps

1.Pick a Beacon Mission
2.Park the sub, sitting on top of the beacon
3.Activate the beacon
4.Attempt to move the sub afterwards

### Bug prevalence

Happens regularly

### Version

Faction test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by Rehexr** [comment]
Huh, certainly didnt teleport the sub on top of the beacon, though my only guess the only thing I could possibly blame otherwise might be the fact that the sub sat on top is using custon hulls to make a moonpool of sorts, which might mess some things up when the beacon is turned on? Ill give it a try for a moment, and see if thats the issue.

**Comment by Rehexr** [comment]
That seems to be it on my end, was able to get it to happen again. Though considering the issue is hull-wise related to having an open hatch and two hulls touching without any kind of gap between them, might be other scenarios where it can be repliacted? Ill probably try it more with similar subs

**Comment by Rehexr** [comment]
Yeah, ive had no luck replicating this in the most recent build, even in what id assume to be the identical situation as here, tried it between 3 subs and beacon stations and no luck, suppose it might just not be an issue anymore for some reason?

**Comment by Regalis11** [comment]
No known steps to repro, closing

**Comment by zsh-is-life** [comment]
Issue still exists on version 1.0.20.1. No mods in use. Singleplayer vanilla.
Typhon 2 ended up being stick to the beacon station's transducer. It some how ended up being wedged into where the bottom right gun is (it was the chaingun in my case). Tried emptying/filling ballasts and tried positive and negative thrust. Submarine was fully powered and operational, except the fact that it could not move at all.
